### PR TITLE
Feature/split endpoints

### PR DESCRIPTION
**Pull request by itsnameissteven**
## Is this a fix or a feature?
Feature
## What is the change?
* An endpoint was added to the backend so logic has been split
* User info is now sent separately from car info
* Tests have been updated and stubbed
## What does it fix?
n/a
## Where should the reviewer start?
Review the Registration component
## How should this be tested?
locally run npm i and sign up as a new user

### CODE REVIEW

File: cypress/integration/hitch-detailed-route_spec.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
+import { createYield } from "typescript";
```
Comment (alexthompson207) [inline cypress/integration/hitch-detailed-route_spec.ts:1]: Curious what this does?

File: src/apiCalls.ts [hunk 2]
```diff
@@ -16,22 +20,72 @@ interface RouteData {
...
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify(carInfo)
+  })
+    .then(handleErrors)
+    .then(() => {
+      setPage(page + 1);
+      setError(false);
+    })
+    .catch(() => setError(true))
```
Comment (alexthompson207) [inline src/apiCalls.ts:66]: This looks great, nice work!

**Review by alexthompson207 — APPROVED** [review]
Nice work with this Steven, this looks great!

File: cypress/integration/hitch-detailed-route_spec.ts (hunk 1 above)
Comment (itsnameissteven) [reply cypress/integration/hitch-detailed-route_spec.ts:1]: imports types for dealing with mocha/ chai types. like 'describe'

Comment (PaigeVannelli) [reply cypress/integration/hitch-detailed-route_spec.ts:1]: Interesting, I haven't seen this before!

File: src/utilities/utilities.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    let hour: string = time?.split(':')[0];
+    let min: string = time?.split(':')[1];
+    let hourToNum = parseInt(hour)
+    if (hourToNum >= 12) {
+      let formattedTime = hourToNum - 12 === 0 ? hourToNum : hourToNum - 12;
+      return `${formattedTime}:${min} pm`
+    } else {
+      return `${hourToNum === 0 ? '12' : hourToNum}:${min} am`
+    }
+  }
```
Comment (PaigeVannelli) [inline src/utilities/utilities.ts:11]: Thanks for working on this!

**Review by PaigeVannelli — APPROVED** [review]

**Pull request merged**